tests/autoblocks/test_run_test_suite.py

```diff
@@ -1800,25 +1799,15 @@ class MyExecutionContext(
         __params_class__ = MyPromptParams
         __template_renderer_class__ = MyTemplateRenderer
 
-    class MyMinorVersion(Enum):
-        v0 = "0"
-        LATEST = "latest"
-
     class PromptManagerA(
-        AutoblocksPromptManager[
-            MyExecutionContext,
-            MyMinorVersion,
-        ],
+        AutoblocksPromptManager[MyExecutionContext,],
```

Comment:
Does the formatter add these commas?

Reply:
no that's black reformatting (it was originally like this):

```python

AutoblocksPromptManager[
  MyExecutionContext,
],